### DISCUSSION TITLE
Spacing in Stringified JSON

### DISCUSSION THREAD

**Discussion by MQumairi** [pocoproject/poco#3788] [General]
Hello!

I'm wondering if there's an efficient way to deal with the formatting of the stringified JSON in version 1.12 of the Poco Libraries, apart from the indent and step. Specifically, I'm looking to add spacing around the colon in the JSON string.

To elaborate, given the following code:
```
Poco::JSON::Object json;
json.set("foo", "bar");
json.stringify(std::cout, 4);
```

Using an older version of the Poco libraries (`libpocojson50`, which I believe corresponds to version 1.8), the following string would be printed to `cout`:
```
{
  "foo" : "bar",
}
```
However, using the most recent version we get:
```
{
  "foo": "bar",
}
```
Is there a way to imitate the behavior of Poco 1.8, and re-apply the space before the colon?

**Comment by aleks-f** [comment]
Currently, no

**Reply by MQumairi** [reply]
Thank you for confirming!
